fusor-ember-cli/app/routes/subscriptions/select-subscriptions.js

```diff
@@ -52,6 +52,10 @@ export default Ember.Route.extend({
                                      'deployment': deployment
                                     });
                  sub.save();
+              } else {
+                // update quantity_attached is it may have changed since record was created
+                matchingSubscription.set('quantity_attached', pool.get('qtyAttached'));
+                matchingSubscription.save();
```

Comment:
This is a bug fix if a subscription management application was somehow updated after the record was created in `Fusor::Subscription`